daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -694,15 +694,6 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite) {
     roundTrip: RoundTrip,
     implString: Option[String]): Unit
 
-  // Provide ability to override existing (default) tunables
-  private def retrieveTunablesCombined(existingTunables: Map[String, String], cfg: DefinedConfig) = {
-    // Note, ++ on Maps replaces any key/value pair from the left with that on the
-    // right, so key/value pairs defined in tunables overrule those defiend in
-    // the config file
-    val combined = existingTunables ++ cfg.tunablesMap
-    combined
-  }
-
```

Comment:
Your PR & commit message doesn't mention this change - is it another cleanup (removing a function that's not used anymore)?